tests/versioning.go

```diff
@@ -91,7 +91,12 @@ func (s *VersioningIntegSuite) SetupSuite() {
 		// versioning data relatively quickly. In general, we only promise to act on new
 		// versioning data "soon", i.e. after a long poll interval. We can reduce the long poll
 		// interval so that we don't have to wait so long.
+		// TODO: update this comment. it may be outdated and/or misleading.
```

Comment:
This is because we don't interrupt long polls on change of versioning data, so if polls from different build ids go to the same matcher, old ones can steal tasks from new ones. If polls from different build ids never go to the same matcher anymore then we don't need it.